src/main/common/string_light.c

```diff
@@ -78,3 +78,14 @@ int strncasecmp(const char * s1, const char * s2, size_t n)
 
     return d;
 }
+
+char *strcasestr(const char *haystack, const char *needle) 
+{
+  int nLen = strlen(needle);
+  do {
+     if(!strncasecmp(haystack, needle, nLen))
```

Comment:
Space between `if` and opening parenthesis.
Also, I think the preferred style in BF was to enclose single-line statements with curly braces

Reply:
Don't like the curly braces when it is intentional to have only single statement ... but you are right